internal/utils/crypto.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "golang.org/x/crypto/bcrypt"
+
+func CheckPassword(pwdHash []byte, pwd []byte) bool {
+	err := bcrypt.CompareHashAndPassword(pwdHash, pwd)
+	if err != nil {
+		return true
+	}
```

Comment:
Since the `bycrpt.CompareHashAndPassword` returns nil on success or an error on failure, shouldn't this `if` statement return `false`?

Reply:
You are correct. That's the wrong way around. Will fix.